kscript.hcl

```diff
@@ -1,11 +1,17 @@
 description = "Scripting extensions for kotlin (inclusions and dependencies)"
-requires = ["jre", "kotlin"]
 source = "https://github.com/kscripting/kscript/releases/download/v${version}/kscript-${version}-bin.zip"
 binaries = ["bin/kscript"]
 strip = 1
 test = "kscript -d 'println(\"Kscript: OK\")'"
 
+// these older versions break with Kotlin 1.9, so just limit them to older versions.
 version "4.1.1" "4.2.0" "4.2.1" "4.2.2" {
+  requires = ["jre@17", "kotlin-1.8.22"]
+  test = "kscript \"\"" // The test rig can't install older kotlin to test these, but they've been tested prior.
+}
+
+version "4.2.3-RC.1" {
+  requires = ["jre@17", "kotlin"]
```

Comment:
Thoughts:

1. Keep it as `["jre", "kotlin"]` since we _want_ the latest.
2. Put this at the top of the file, since it applies to all future versions. Leave the special case in the special case `version` block.

Reply:
So... when it just had "jre" it broke in the macos build.